### ISSUE TITLE
what have you done

### ISSUE THREAD

**Issue by zorbulator** [ScanuNicco/Boat-Race#41]
![image](https://user-images.githubusercontent.com/25832911/53592855-07725f00-3b4c-11e9-9d5e-ab0a5ec11e08.png)

**Comment by ScanuNicco** [comment]
I don't think that this is my fault, it was working last time I committed it. My last two commits didn't even change the code. Re exporting the obj also had no affect on this.

**Comment by zorbulator** [comment]
It seems like it might have something to do with the sky and the code for the sky might have changed. We could try to find new sky code, fix the problem with the existing sky, or try to get the code that was working.

**Comment by zorbulator** [comment]
It's actually a problem with the water so I removed it until we can fix this
